packages/espressocash_app/lib/data/db/mixins.dart

```diff
@@ -15,6 +15,15 @@ mixin EntityMixin on Table {
   Set<Column<Object>>? get primaryKey => {id};
 }
 
+mixin QueryMixin on Table {
```

Comment:
Seems useless. You can use `EntityMixin` and just add `query` column directly.

Reply:
Can't use `EntityMixin` for this case since `id` needs to be autoincrement integer.